### ISSUE TITLE
Config file not found

### ISSUE THREAD

**Issue by finnure** [anvilco/spectaql#257]
When running `spectaql apidocs.config.yml` using version 1.0.0 I now get an error saying apidocs.config.yml does not exists. spectaql expects the config file to be in the node_moduesl/spectaql folder instead of the project root folder. 

![image](https://user-images.githubusercontent.com/89914707/161249953-71333e20-0ac5-4de5-bf89-b5e85eaae5b7.png)

**Comment by newhouse** [comment]
Ugh, sorry, will fix. For now, if you put a full path to your file, things should work:
`spectaql /full/path/to/apidocs.config.yml`

**Comment by newhouse** [comment]
@finnure @mxseev

Please try again with `1.0.2` and let me know if things are working?

**Comment by mxseev** [comment]
Yup, but another bug there

```
~/D/P/-api ❯❯❯ cat package.json | grep spectaql                                                             
    "build:docs": "spectaql spectaql.yaml -t public/docs",
    "spectaql": "^1.0.2",
~/D/P/-api ❯❯❯ npm run build:docs                                                                            

> -api@0.9.1 build:docs
> spectaql spectaql.yaml -t public/docs

node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module './dist'
Require stack:
- /Users/mxseev/Documents/Projects/-api/node_modules/spectaql/index.js
- /Users/mxseev/Documents/Projects/-api/node_modules/spectaql/bin/spectaql.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/mxseev/Documents/Projects/-api/node_modules/spectaql/index.js:2:58)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/mxseev/Documents/Projects/-api/node_modules/spectaql/index.js',
    '/Users/mxseev/Documents/Projects/-api/node_modules/spectaql/bin/spectaql.js'
  ]
}
~/D/P/-api ❯❯❯
```

**Comment by newhouse** [comment]
@mxseev

Double-ugh. Bad publish. Please try `1.0.3`?!

**Comment by mxseev** [comment]
Is glob support for input files removed in 1.0?

```
~/D/P/-api ❯❯❯ cat package.json | grep spectaql                                                             ✘ 1 main ✭ ✱
    "build:docs": "spectaql spectaql.yaml -t public/docs",
    "spectaql": "^1.0.3",
~/D/P/-api ❯❯❯ npm run build:docs                                                                               main ✭ ✱

> -api@0.9.1 build:docs
> spectaql spectaql.yaml -t public/docs

/Users/mxseev/Documents/Projects/-api/node_modules/spectaql/dist/spectaql/graphql-loaders.js:53
      throw new Error(`GraphQL schema file does not exist at ${path}`);
      ^

Error: GraphQL schema file does not exist at /Users/mxseev/Documents/Projects/-api/src/schema/*.graphql
```

**Comment by newhouse** [comment]
@mxseev fixed in `1.0.4`.

**Comment by mxseev** [comment]
And everything is working great!
Thank you very much, it was a pleasure to participate.
